extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -241,6 +233,19 @@ export class SchemaCompareDialog {
 		}
 	}
 
+	private getEndpointName(endpointType: mssql.SchemaCompareEndpointType): string {
```

Comment:
it's a little confusing that there's this `getEndpointName()` defined here and another `getEndpointName()` in utils.cs that does something different and is used below in `endpointChanged()`

Reply:
Ah, good catch.  Didn't notice that before.